packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -81,7 +81,7 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
             rateLimiter,
             resolvedUrl.endpoints.ordererUrl,
         );
-        await ordererRestWrapper.post(
+        const containerId = await ordererRestWrapper.post<string>(
```

Comment:
Are we actually creating docId here instead of container id? ContainerId is just the id for a particular instance of docId. Sp we can have many containerIds for a given docId.

Reply:
good point. We should just documentId instead of containerId?

Reply:
Yes, I think it should be called docId.

Reply:
Ah! Good to know, I've always wondered about the difference. Will update